locales/FR_fr.json

```diff
@@ -0,0 +1,141 @@
+			},
+			{
+				"question": "Aurais-je besoin de modifier ma console pour me connecter ?",
+				"answer": "Oui, vous devrez modifier votre appareil afin de vous connecter; cependant sur Wii U un accès au Homebrew Launcher est suffisant (avec Haxchi, Coldboot Haxchi, ou même la faille du Navigateur Web). La démarche à suivre pour la 3DS sera fournie ultérieurement."
+			}
+		]
+	},
+	"credits": {
+		"title": "L'équipe",
+		"text": "Rencontrez l'équipe derrière le projet",
+		"people": [
+			{
+				"name": "Jonathan Barrow (jonbarrow)",
+				"caption": "Chef de projet et développeur principal",
+				"picture": "https://github.com/jonbarrow.png",
+				"github": "https://github.com/jonbarrow"
+			},
+			{
+				"name": "Jemma (CaramelKat)",
+				"caption": "Étude et développement sur Miiverse",
+				"picture": "https://github.com/caramelkat.png",
+				"github": "https://github.com/CaramelKat"
+			},
+			{
+				"name": "superwhiskers",
+				"caption": "Développement de la bibliothèque crunch",
+				"picture": "https://github.com/superwhiskers.png",
+				"github": "https://github.com/superwhiskers"
+			},
+			{
+				"name": "SuperMarioDaBom",
+				"caption": "Étude des consoles et d'autres systèmes",
+				"picture": "https://github.com/supermariodabom.png",
+				"github": "https://github.com/SuperMarioDaBom"
+			},
+			{
+				"name": "Rambo6Glaz",
+				"caption": "Installateur du réseau et études des consoles",
+				"picture": "https://github.com/Rambo6Glaz.png",
+				"github": "https://github.com/Rambo6Glaz"
+			},
+			{
+				"name": "mrjvs",
+				"caption": "Développement Web et premières études de l'eShop",
+				"picture": "https://github.com/mrjvs.png",
+				"github": "https://github.com/mrjvs"
+			},
+			{
+				"name": "Benjamin Moss (kxpler442b)",
+				"caption": "Developpement Web",
+				"picture": "https://github.com/kxpler442b.png",
+				"github": "https://github.com/kxpler442b"
+			},
+			{
+				"name": "Jip Fr",
+				"caption": "Developpement Web",
+				"picture": "https://github.com/jipfr.png",
+				"github": "https://github.com/jipfr"
+			},
+			{
+				"name": "Shutterbug2000",
+				"caption": "Étude de Mario Kart 7 et de la 3DS",
+				"picture": "https://cdn.discordapp.com/avatars/191370953807233024/0311b61e2009c1576828dd2e9a59d72e.png?size=128",
+				"github": "https://github.com/shutterbug2000"
+			},
+			{
+				"name": "Kinnay",
+				"special": "Remerciement spéciaux",
+				"caption": "Étude des structures de données de Nintendo",
+				"picture": "https://cdn.discordapp.com/avatars/186572995848830987/b55c0d4e7bfd792edf0689f83a25d8ea.png?size=128",
+				"github": "https://github.com/Kinnay"
+			}
+		]
+	},
+	"progressPage": {
+		"title": "Notre progression",
+		"description": "Vérifiez l'avancement et les objectifs du projet ! (Mis à jour environ toute les heures, ne reflète pas TOUS nos objectifs et progression)."
```

Comment:
```suggestion
		"description": "Vérifiez l'avancement et les objectifs du projet! (Mis à jour environ toute les heures, ne reflète pas TOUS nos objectifs et notre progression)."
```

Reply:
According to this website https://grammaire.reverso.net/les-espaces-et-la-ponctuation/, there should always be a space before and after exclamation and questions marks

Reply:
however the part about the missing "notre" seems right, i don't know if it's possible to partially commit a suggestion ?

Reply:
Thanks for pointing that out! Fixed the suggestion so that it only corrects notre.